### DISCUSSION TITLE
Blog posts, Talks and Podcasts that feature Uno

### DISCUSSION THREAD

**Discussion by ghuntley** [unoplatform/uno#5013] [General]
👋 Howdy folks, if you see a blog post, talk or podcast that features Uno please share (even if you are the author of the content) below as a comment. 👇

# Blog Posts

## About Uno

- https://hackernoon.com/cross-platform-mobile-apps-with-net-and-uno-dee2b024281d
- https://www.freecodecamp.org/news/uno-platform/

## Getting Started with Uno

- https://keyholesoftware.com/2019/08/12/using-c-xaml-uno-platform-to-build-one-codebase-cross-platform-apps/
- https://nicksnettravels.builttoroam.com/post-2019-03-10-getting-started-with-platform-uno-aspx/
- https://nicksnettravels.builttoroam.com/post-2018-06-15-building-a-tipcalc-using-platform-uno-aspx/

## Using Uno

- https://nicksnettravels.builttoroam.com/uwp-splitview/
- https://nicksnettravels.builttoroam.com/uno-as-a-pwa/

## Libaries

- http://www.lhotka.net/weblog/UnoPlatformAndWebAssemblyWithCSLAV5.aspx

## Contributing to Uno

- https://blog.mzikmund.com/2019/03/contributing-to-the-uno-platform-part-1/
- https://blog.mzikmund.com/2019/04/contributing-to-the-uno-platform-part-2/
- https://blog.mzikmund.com/2019/04/contributing-to-uno-platform-part-3/

# Xamarin Forms on Webassembly

- https://javiersuarezruiz.wordpress.com/2019/10/16/xamarin-forms-en-la-web-con-webassembly-y-uno/

# UnoConf

- https://blog.mzikmund.com/2019/09/unoconf-2019/

# Podcasts

> On this episode of Eat Sleep Code, Jérôme Laban, Uno Platform CTO talks about the prospects of using Uno to create cross platform .NET applications. Jérôme also discusses the use of WebAssembly by Uno and what WebAssembly means for deploying .NET to the web.
https://www.stitcher.com/podcast/eat-sleep-code-the-official-telerik-podcast/e/60719171

- [Herding Code 239: Jerome Laban on Uno Platform
](https://herdingcode.com/herding-code-239-jerome-laban-on-uno-platform/)

> With Uno, you can build native apps and Web Assembly apps using WinRT (UWP) APIs. In this episode, Daniel is talking with Jérôme Laban, the CTO of Uno Platform about what Uno is, the history of Uno, the current state of Uno and much more. 
- [App In The Cloud • Episode 6](https://share.transistor.fm/s/6fe1a0c2)

# Talks

- [Jérôme Laban at Xamarin Developer Summit on the future of WebAssembly and .NET](https://youtu.be/t1cQsenAmNo?t=1049)
- [Microsoft Build: Building WebAssembly apps with Microsoft's stack - Lessons from the trenches](https://www.youtube.com/watch?v=mhoEqkJQlkY)
- [Meet the Uno Platform UWP everywhere | Martin Zikmund | MSP Czech & Slovakia](https://www.youtube.com/watch?v=NHFbPItDaDc)

**Comment by MartinZikmund** [comment]
I had a live webinar on Uno for Microsoft Student Partners - https://youtu.be/NHFbPItDaDc

<blockquote><img src="https://i.ytimg.com/vi/NHFbPItDaDc/maxresdefault.jpg" width="48" align="right"><div><img src="/yts/img/favicon_144-vfliLAfaB.png" height="14"> YouTube</div><div><strong><a href="https://www.youtube.com/watch?v=NHFbPItDaDc">Meet the Uno Platform UWP everywhere | Martin Zikmund | MSP Czech & Slovakia</a></strong></div><div>UWP is a wonderful app development platform, which allows targeting all Windows devices at once with a single codebase. But what if there was a framework, th...</div></blockquote>

**Comment by ghuntley** [comment]
@all-contributors please add @rockfordlhotka for his blog at http://www.lhotka.net/weblog/UnoPlatformAndWebAssemblyWithCSLAV5.aspx

<blockquote><div><strong><a href="http://www.lhotka.net/weblog/UnoPlatformAndWebAssemblyWithCSLAV5.aspx">Rockford Lhotka - Uno Platform and WebAssembly with CSLA v5</a></strong></div></blockquote>

**Comment by ghuntley** [comment]
@all-contributors please add @nickrandolph for his blog at https://nicksnettravels.builttoroam.com/uno-as-a-pwa/

<blockquote><img src="https://blogimages.builttoroam.com/2019-08-uno-as-a-pwa/05.png" width="48" align="right"><div>Nick's .NET Travels</div><div><strong><a href="https://nicksnettravels.builttoroam.com/uno-as-a-pwa/">Installing Uno as a PWA with WebAssembly - Nick's .NET Travels</a></strong></div><div>One of the things that I don’t like about the web is that I have to access everything via the browser. Whilst favourites and bookmarks are all well and good, one of the nice things about apps is that I can do things like pin to start/taskbar (Windows) or add to homescreen (Android). Sure I … Continue reading "Installing Uno as a PWA with WebAssembly"</div></blockquote>

**Comment by mizzleinetimi** [comment]
https://www.freecodecamp.org/news/uno-platform/

<blockquote><img src="https://www.freecodecamp.org/news/content/images/2019/10/UnoLogoLargeCut-1.png" width="48" align="right"><div>freeCodeCamp.org</div><div><strong><a href="https://www.freecodecamp.org/news/uno-platform/">Uno - One Platform to Rule Them All</a></strong></div><div>First, we should start off with what Uno is and why you should care. 

As stated on their website [https://platform.uno], Uno is "The only platform for
building native mobile, desktop and WebAssembly apps with C#, XAML from a single
codebase. Open source and professionally supported." 

What does this mean?
Well if you have any experience (or not) building a mobile app and a subsequent
web app, you have essentially had to build both separately, outside of
potentially sharing data through an API.</div></blockquote>

**Comment by ghuntley** [comment]
Xamarin Forms on WebAssembly — https://javiersuarezruiz.wordpress.com/2019/10/16/xamarin-forms-en-la-web-con-webassembly-y-uno/

<blockquote><img src="https://javiersuarezruiz.files.wordpress.com/2019/10/wasm-project.png" width="48" align="right"><div><img src="https://s1.wp.com/i/favicon.ico" height="14"> Javier Suárez | Blog</div><div><strong><a href="https://javiersuarezruiz.wordpress.com/2019/10/16/xamarin-forms-en-la-web-con-webassembly-y-uno/">Xamarin.Forms en la web con WebAssembly y UNO</a></strong></div><div>Introducción Xamarin.Forms en la web. No es la primera vez que hablamos de ello. Dependiendo de con quien hables es una idea que trae malos recuerdos y piensa en algo horrible o piensa que es genia…</div></blockquote>

**Comment by kaki104** [comment]
This is a YouTube video provided in Korean.

Creating a simple RSS reader using Uno Platform
Part1 : youtu.be/CQBfyTSV5uc
Part2 : youtu.be/WVN1SgKOGvI

What is UNO Platform?
youtu.be/li47OqE0tNI
